pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/SegmentPreProcessor.java

```diff
@@ -158,8 +170,17 @@ public void process(@Nullable SegmentOperationsThrottler segmentOperationsThrott
     // that the other required indices (e.g. forward index) are up-to-date.
     try (SegmentDirectory.Writer segmentWriter = _segmentDirectory.createWriter()) {
       if (processStarTrees(indexDir, segmentOperationsThrottler)) {
-        // NOTE: When adding new steps after this, un-comment the next line.
```

Comment:
Should we keep this comment, moved to line 183 so we don't forget to reload metadata if we add more steps?